src/index.html

```diff
@@ -7,8 +7,32 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap"  rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="logo"
+         src="images/logo.png"
+         alt="moyo">
+      </a>
+      <nav>
+        <ul class="nav">
```

Comment:
<img width="1026" alt="Screenshot 2023-08-13 at 15 38 31" src="https://github.com/mate-academy/layout_moyo-header/assets/135232001/4bc587b9-161f-4ba6-8983-49e785474f32">
<img width="1067" alt="Screenshot 2023-08-13 at 15 38 40" src="https://github.com/mate-academy/layout_moyo-header/assets/135232001/7a9bbe00-64d9-40fc-a0d6-f8ea47aedcc1">
There are a bunch of validation errors, consider fixing them